types/aws-lambda/index.d.ts

```diff
@@ -35,6 +35,7 @@
 //                 Alexander Pepper <https://github.com/apepper>
 //                 Alessandro Palumbo <https://github.com/apalumbo>
 //                 Sachin Shekhar <https://github.com/SachinShekhar>
+//                 Ivan Martos <https://github.com/ivanmartos>
```

Comment:
Just to check: you are aware this means you will be pinged for every PR on this package going forward?

Reply:
yes I am - that's what community is about :)

Reply:
👍

Reply:
👍